app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @rooms.each do |room| %>
+  <%= room.suite %>
+  <%= room.rate %>
+  <%= room.hotel.name %>
+  
+<% end %>
```

Comment:
We also want to see on this page, the number of guests that have stayed in each room.